tests/Integration/Behaviour/Features/Scenario/CatalogPriceRule/catalog_price_rules_listing.feature

```diff
@@ -0,0 +1,305 @@
+      | reduction value | 111.50               |
+      | shop            | testShop             |
+      | includes tax    | true                 |
+      | price           | 50                   |
+    Then catalog price rule "catalogPriceRuleReference1" should have following details:
+      | catalog price rule detail | value                |
+      | name                      | catalog price rule 1 |
+      | currency                  | usd                  |
+      | country                   | UnitedStates         |
+      | group                     | visitor              |
+      | from quantity             | 1                    |
+      | reduction type            | amount               |
+      | reduction value           | 111.50               |
+      | shop                      | testShop             |
+      | includes tax              | true                 |
+      | price                     | 50                   |
+    When I add catalog price rule "catalogPriceRuleReference2" with following details:
+      | name            | catalog price rule 2  |
+      | currency        | usd                   |
+      | country         | UnitedStates          |
+      | group           | visitor               |
+      | from quantity   | 1                     |
+      | reduction type  | amount                |
+      | reduction value | 50                    |
+      | shop            | testShop              |
+      | includes tax    | true                  |
+      | price           | 10                    |
+    Then catalog price rule "catalogPriceRuleReference2" should have following details:
+      | catalog price rule detail | value                |
+      | name                      | catalog price rule 2 |
+      | currency                  | usd                  |
+      | country                   | UnitedStates         |
+      | group                     | visitor              |
+      | from quantity             | 1                    |
+      | reduction type            | amount               |
+      | reduction value           | 50                   |
+      | shop                      | testShop             |
+      | includes tax              | true                 |
+      | price                     | 10                   |
+    When I add catalog price rule "catalogPriceRuleReference3" with following details:
+      | name            | catalog price rule 3  |
+      | currency        | usd                   |
+      | country         | UnitedStates          |
+      | group           | visitor               |
+      | from quantity   | 5                     |
+      | reduction type  | amount                |
+      | reduction value | 10                    |
+      | shop            | testShop              |
+      | includes tax    | true                  |
+      | price           | 20                    |
+    When I add catalog price rule "catalogPriceRuleReference4" with following details:
```

Comment:
```suggestion
    And I add catalog price rule "catalogPriceRuleReference4" with following details:
```

I see you got tired of checking the context of each rule after its creation 😅 You're right it should be enough if two have already been checked